src/sparseml/export/export.py

```diff
@@ -46,8 +46,9 @@ def export(
     single_graph_file: bool = True,
     num_export_samples: int = 0,
     batch_size: int = 1,
+    recipe: Optional[Union[Path, str]] = None,
     deployment_directory_name: str = "deployment",
-    device: str = "auto",
+    device: str = "cpu",
```

Comment:
Why do we change this default from auto to cpu? on line 118 it looks like we're expecting auto as a default